buổi-10/main.js

```diff
@@ -0,0 +1,150 @@
+// bài 1
+const getMultiplication = number => {
+    if (number >= 1 && number <= 10) {
+        for(let i = 1; i <= 10; i++) {
+            console.log(number + ' * ' + i + ' = ' + (number * i))
+        } 
+    } else {
+        console.log('Chỉ nhận giá trị từ 1-> 10!!')
+    }
+}
+
+// bài 2
+const factorial = number => {
+    if(number === 0){
+        return 1
+    }
+    return number * factorial(number - 1)
+}
+const Result = number => {
+    if(number >= 1 && number <= 30) {
+       return factorial(number)
+    } else {
+       console.log('Không thỏa mãn điều kiện của n!!')
+   }
```

Comment:
Khi viết thì hạn chế các block lồng nhau, hạn chế viết `if else`
Cách viết sau đây nhìn sẽ gọn hơn

```suggestion
  if (number < 1 || number > 30) return;
  
  return factorial(number)
```